lectures/markov_asset.md

```diff
@@ -470,7 +470,7 @@ m_{t+1} = \beta \frac{u'(c_{t+1})}{u'(c_t)}
 
 where $u$ is a concave utility function and $c_t$ is time $t$ consumption of a representative consumer.
 
-(A derivation of this expression is given in a [later lecture](https://python-advanced.quantecon.org/lucas_model.html))
```

Comment:
@HumphreyYang this lecture has been moved to `jax` so we will need to update this link.